# Request
string param_name
bool new_value
---
# Response
bool success
string message